community/governance.md

```diff
@@ -0,0 +1,376 @@
+  candidate for a dynamic charter. Alternatively, language-based
+  maintenance teams also have a dynamic charter.
+- A *static* charter means that all membership decisions and non-trivial
+  policy changes must be approved by the core team. For example, a
+  finance team may require a static charter.
+
+All sub-teams must adhere to the governance, policies, and procedures of
+conda-forge at all times.
+
+## Voting
+
+This section presents descriptions and criteria for voting items in the
+conda-forge community. The core team is the only team with voting
+rights. Members of core may also call a vote on any topic. The
+restrictions on calling a vote are as follows:
+
+- There must only be one vote active on a particular item at any time.
+- The act of calling for a vote cannot itself violate the code of
+  conduct. For example, Sam repeatedly called for votes immediately
+  after a previous vote failed to achieve Sam's result. Sam is
+  attempting to bully other members of core into agreeing, and is thus
+  violating the code of conduct.
+- Voting yes moves the proposal forward; voting no is the only way to
+  express opposition to the proposal; not voting is discouraged, but
+  non-votes do not count as "no".
+- There should always be an option to abstain from voting.
+
+Voting items are labeled as either **standard** or **sensitive**.
+Standard items are ones where public record and discourse is preferable.
+Sensitive voting items are ones where the results of the vote should
+remain private to the voters after the vote has occurred. Sensitive
+votes should take place on a secure anonymous voting platform in order
+to retain election integrity and anonymity. (We have used
+[Polys](https://polys.me) and [the Helios voting
+system](https://vote.heliosvoting.org/), but are open to any secure,
+anonymous system.) The email capability of your chosen voting platform
+should be used for sending voting invitations and reminders, and you
+should use the email list from
+<https://github.com/conda-forge/conda-forge.github.io/blob/main/src/core.csv>
+and
+<https://github.com/conda-forge/conda-forge.github.io/blob/main/src/emeritus.csv>
+as the authoritative list of emails to use.
+
+The default voting period is 1 week (7 days). This may be modified at
+the time when a vote is called, but may never be less than 24 hrs.
+
+Additional requirements may apply in case low turnouts have to be
+handled, see "Quorum" below.
+
+To call for a standard vote, here is a template PR comment:
+
+``` md
+@conda-forge/core
+This PR falls under the {policy} policy, please vote and/or comment on this PR.
+This PR needs {policy_percent} of core to vote yea to pass.
+To vote please leave Approve (yea) or Request Changes (nay) reviews.
+If you would like changes to the current language please leave a comment or push to this branch.
+This vote will end on {date}.
+```
+
+------------------------------------------------------------------------
+
+Posting results  
```

Comment:
```suggestion
### Posting results 
 
```

And the same for the below subheaders till we arrive to `## Current Members of Core`.

Reply:
👍🏼